javascript/frameworks/cap/lib/advanced_security/javascript/frameworks/cap/CDS.qll

```diff
@@ -555,16 +586,30 @@ class CdsUser extends API::Node {
   }
 }
 
-class CdsTransaction extends MethodCallNode {
+class CdsTransaction extends SourceNode {
   ServiceInstance srv;
-
-  CdsTransaction() { this = srv.getAMemberCall("tx") }
+  CallNode txCall;
+
+  CdsTransaction() {
+    txCall = srv.getAMemberCall("tx") and
```

Comment:
This charpred either selects the tx call or the callback passed to the tx call - can you provide a quick example in the doc between the two? It's been very helpful where you've done that else where in the code.

Reply:
Addressed in [a3efdd7](https://github.com/advanced-security/codeql-sap-js/pull/200/commits/a3efdd7058ec5a9b52e577001105c1585336d8d7).